infection.json.dist

```diff
@@ -10,6 +10,12 @@
             "branch": "master"
         }
     },
+    "mutators": {
+        "@default": true,
+        "global-ignoreSourceCodeByRegex": [
```

Comment:
Shouldn't be ignored: alternatively, we can use `roave/infection-static-analysis-plugin`, which should raise mutation score by a lot.

Reply:
So, I've narrowed the ignored code to be specific for the case.
But I don't have enough energy to learn `roave/infection-static-analysis-plugin` in the next dozen of days.
What I can say is that the `assert()` line is only needed to make Psalm happy: the finger should be pointed at `lcobucci/jwt` which doesn't `@psalm` annotate well enough its `Parser` implementation, see

https://github.com/lcobucci/jwt/blob/2f533837091d0b76a89a059e7ed2b2732b2f459e/src/Token/Parser.php#L26-L37

Reply:
Let's roll with this for now :+1: 

As for `roave/infection-static-analysis-plugin`, it is a drop-in replacement for `vendor/bin/infection` that behaves the exact same way.

I'll raise a new issue separately, or perhaps introduce it myself when I got time to tinker around this again :+1:

Reply:
Ok, on my way to the rebase